meson.build

```diff
@@ -403,13 +404,29 @@ if sys_windows or get_option('lua-interpreter') == 'lua'
       break
     endif
   endforeach
+  if not sys_windows # FIXME: Get a correct Lua installation on CI or fix meson.build
```

Comment:
What do you mean by a `meson.build` fix?

Reply:
Ok I started typing things thinking that you were asking because of the "FIXME", then I saw the end of the line

For short: it was to be something like "fix Lua wrap"

Reply:
oh ok